p3/plot/backend/textemplates/navchart.tex

```diff
@@ -0,0 +1,95 @@
+\documentclass{standalone}
+\usepackage[dvipsnames]{xcolor}
+\usepackage{pgfplots}
+
+\pgfplotsset{compat=1.18}
```

Comment:
Why do navchart and cascade use different `compat` values?

Reply:
Largely an artefact of writing the two plot templates independently. I'll check that a single compat value works for both (1.18 is probably fine for both), and then update for consistency